frontend/cypress/tests/zoomView/zoomView_hideCommitMessages.js

```diff
@@ -53,4 +53,174 @@ describe('hide all commit messages ', () => {
     cy.get('#tab-zoom .commit-message .body')
         .should('be.visible');
   });
+
+  it('should only display hide all commit messages when none are hidden', () => {
+    // open the commit panel
+    cy.get('.icon-button.fa-list-ul')
+        .should('be.visible')
+        .first()
+        .click();
+
+    cy.get('#tab-zoom .commit-message', { timeout: 90000 })
+        .should('be.visible');
+
+    // should only display 'hide all commit messages'
+    cy.get('#tab-zoom .toolbar--multiline')
+        .children()
+        .should('have.length', 1);
+
```

Comment:
I think you can also try clicking it and verify that it indeed hides the commits as intended. Similar for the show case

Reply:
Oh I thought it was already covered in the first test case in line 2? Or would it be better to combine the test cases and check the text & functionality together?

Reply:
Thanks for noting that. You can ignore the previous comment